gunicorn/workers/ggevent.py

```diff
@@ -10,20 +10,19 @@
 from functools import partial
 import time
 
-_socket = __import__("socket")
-
-# workaround on osx, disable kqueue
-if sys.platform == "darwin":
-    os.environ['EVENT_NOKQUEUE'] = "1"
-
 try:
     import gevent
 except ImportError:
     raise RuntimeError("You need gevent installed to use this worker.")
+
+# validate the gevent version
+if gevent.version_info < (1, 2, 0):
```

Comment:
Version 1.2.0 is when subprocess support really stabilized and compatibility for Python 3.5 and 3.6 was added.[1](http://www.gevent.org/whatsnew_1_2.html).

Reply:
Version 1.2 was also when `gevent.version_info` was deprecated 😄 (It doesn't always get updated because it's not a process automated by zest.releaser.) The suggested alternative is  to use `pkg_resources.parse_version(gevent.__version__)` (or the equivalent `packaging.version.Version(gevent.__version__)`. You could also query the `pkg_resources` working set.

Reply:
Version 1.4 is the first to fully support Python 3.7.

Personally, I don't bother much with 1.2 anymore. The only thing it has going for it over 1.3 is "official" support for Python 2.7 < 2.7.9. gevent 1.2 didn't support Python 2.6.

If I understand correctly, `master` of gunicorn is now Python 3 only, so I don't really see a reason not to require the latest gevent, gevent 1.4.

Other than the minor comments, this seems fine to me.

Reply:
Have you encountered any incompatibilities in the wild with third party libraries that would prevent users from upgrading to 1.4?

Reply:
Is there a chance you could use one of the other options instead of accessing `gevent.version_info`, since that is deprecated?

Reply:
Thank you!